src/modules/ekf2/EKF/common.h

```diff
@@ -334,6 +334,7 @@ struct parameters {
 	// these parameters control the strictness of GPS quality checks used to determine if the GPS is
 	// good enough to set a local origin and commence aiding
 	int32_t gps_check_mask{21};             ///< bitmask used to control which GPS quality checks are used
+	int32_t req_fix{3};                     ///< maximum acceptable horizontal position error (m)
```

Comment:
Comment needs to be updated.

Reply:
Thank you you're right, I forgot it after copy/paste